lib/ravelin/event.rb

```diff
@@ -39,7 +39,7 @@ def object_classes
     private
 
     def validate_top_level_payload_params
-      validate_customer_presence_on :order, :items, :paymentmethod,
+      validate_customer_id_presence_on :customer, :order, :paymentmethod,
```

Comment:
We don't want to validate the presence of a root `:customer_id` or `:temp_customer_id` on the customer event.